superduperdb/__main__.py

```diff
@@ -2,8 +2,10 @@
 
 import click
 
-from .cli import app, config, docs, info  # noqa: F401, E402
-from .cli.serve import local_cluster, serve  # noqa: F401, E402
+from .cli import app, config, docs, info
+from .cli.serve import local_cluster, serve
+
+__all__ = 'config', 'docs', 'info', 'local_cluster', 'serve'
```

Comment:
The short story is that it prevents the linter from removing these imports and breaking our code, something the `noqa` did before, since I wanted to completely disallow these and this was the only half-way decent use.

The literal meaning of `__all__` is exactly "what is imported with `import *`: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

But that's not so interesting.  Almost no one does `import *`, and in a lot of code, and to mypy and our linters, it has a secondary meaning as "the symbols you are exporting outside the module".

Otherwise, each of these has to have two noqas, one to tell black not to clean out the import, and one to tell ruff not to complain about the unused import.

----

I mean, you can also point out that exactly these symbols are in fact being exposed beyond this module, so it's not really an abuse.